pkg/logging/log.go

```diff
@@ -44,13 +51,18 @@ func Setup(logFormat string) error {
 		// in text mode we use FormatSerialize format
 		globalLog = klogr.New()
 	case JSONFormat:
-		zapLog, err := zap.NewProduction()
-		if err != nil {
-			return err
-		}
-		klog.SetLogger(zapr.NewLogger(zapLog))
+		// zapLog, err := zap.NewProduction()
+		// if err != nil {
+		// 	return err
+		// }
+		// klog.SetLogger(zapr.NewLogger(zapLog))
+		zapLog := logzap.New(logzap.Encoder(&logzap.KubeAwareEncoder{
+			Encoder: zapcore.NewJSONEncoder(setupzap().EncoderConfig),
+			Verbose: false,
+		}))
 		// in json mode we use FormatKlog format
-		globalLog = klog.NewKlogr()
+		// globalLog = klog.NewKlogr()
```

Comment:
klog is not initialised anymore, please address this in another PR

Reply:
noted